{"name": "Проверка невозможности принять заказ при некорректном ID заказа", "status": "passed", "start": 1732976695363, "stop": 1732976695540, "uuid": "45969bb2-08fb-4561-8288-110cb0cc3d6e", "historyId": "98ac9120177e9e9034ab0d99562f962d", "testCaseId": "98ac9120177e9e9034ab0d99562f962d", "fullName": "tests.test_take_order.TestTakeOrder#test_take_order_uncorrect_order_id_error", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_take_order"}, {"name": "subSuite", "value": "TestTakeOrder"}, {"name": "host", "value": "DESKTOP-4GFTOQP"}, {"name": "thread", "value": "10124-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_take_order"}]}